### DISCUSSION TITLE
dark logo is not working

### DISCUSSION THREAD

**Discussion by sparatko242** [imfing/hextra#289] [Q&A]
I think that following line in `layouts/partials/navbar.html` 

    {{- $logoDarkPath := .Site.Params.navbar.logo.dark | default $logoPath -}}

should be like this

    {{- $logoDarkPath := .Site.Params.navbar.logo.dark | default "images/logo-dark.svg" -}}

**Comment by imfing** [comment]
Hi, may i ask how did you configure the dark logo?

**Comment by sparatko242** [comment]
I did put logo.svg and logo-dark.svg to `static/images`

light version is working ok but dark one not

**Comment by imfing** [comment]
can you help provide an example minimal repository that reproduce this issue?

**Comment by zse4t7i** [comment]
My [repos](https://github.com/librehugohu/librehugohu.github.io) has the same problem.

**Comment by sparatko242** [comment]
sorry for late response

I did solve this by renaming logo file to `bm.svg` and `bm-dark.svg` and update configuration

    params:
      navbar:
        displayTitle: true
        displayLogo: true
        logo:
          path: images/bm.svg
          dark: images/bm-dark.svg
          width: 50
          height: 50

**Comment by imfing** [comment]
@librehugohu thanks for providing your repo as an example. I'll have a look at it.

**Comment by zse4t7i** [comment]
> @librehugohu thanks for providing your repo as an example. I'll have a look at it. 

I tried the solution above, and it worked now. You'll see this in my latest commit. And the doc seems should be updated?

**Comment by imfing** [comment]
I still couldn't reproduce this on my instance. 

```
params:
  navbar:
    displayTitle: true
    displayLogo: true
    logo:
      path: images/logo.svg
      dark: images/logo-dark.svg
      link: /
      width: 40
      height: 20
```
![image](https://github.com/imfing/hextra/assets/5097752/110871d2-f98b-4130-8da0-af49f4191ef7)

**Comment by sparatko242** [comment]
but if you remove these two lines from config :
    
    path: images/logo.svg
    dark: images/logo-dark.svg

then defaults file names should be used and it that case it is not working

**Reply by imfing** [reply]
@trieska got it. Could you open a quick PR? Also I think we should fallback to `images/logo-dark.svg` only if it exists, otherwise use the default non-dark logo path. 🤔
